snowflake/data_source.go

```diff
@@ -36,8 +38,14 @@ type DataSourceSyncer struct {
 	repoProvider func(params map[string]string, role string) (dataSourceRepository, error)
 	SfSyncRole   string
 
-	startFrom       string
-	excludeChildren []string
+	startFrom         string
+	excludeChildren   []string
+	skipColumns       bool
+	schemaExcludes    set.Set[string]
+	sharesMap         map[string]struct{}
```

Comment:
I now this is legacy, but maybe we should replace this also by a `set.Set[string]` type